ush/config_defaults.sh

```diff
@@ -691,9 +693,9 @@ ens_h_radardbz=4.10790         #horizontal localization scale of "Gaussian funct
 ens_v_radardbz=-0.30125        #vertical localization scale of "Gaussian function=exp(-0.5)" for radardbz EnVar (positive:grids, negative:lnp)
 nsclgrp=1
 ngvarloc=1
-i_ensloccov4tim=0
-i_ensloccov4var=0
-i_ensloccov4scl=0
+r_ensloccov4tim=1.0
+r_ensloccov4var=1.0
+r_ensloccov4scl=1.0
```

Comment:
@shoyokota This is not a question on this PR but a general question. Do we have the functionality to apply cross-time covariance in the current GSI? If yes, could you explain more about this? Thanks!

Reply:
The cross-time covarianve is effective only in using 4DEnVar (l4densvar=true), so the current RRFS cannot activate it. In 4DEnVar, cross-time covariance between analysis time and the other time bins in the assimilation window is used in the ensemble B. If we use "l_timloc_opt=.true." and make "r_ensloccov4tim" less than one, the impact from observations at different time from the analysis time becomes smaller.

Reply:
Thanks for your answers on this.